docs/book/basic-usage.md

```diff
@@ -30,10 +30,10 @@ class AlbumController extends AbstractActionController
 
 ## Render a Flash Message
 
-Render all flash messages in a view script, e.g. `module/Album/view/album/album/index.phtml`:
+Render success type of flash messages in a view script, e.g. `module/Album/view/album/album/index.phtml`:
 
 ```php
-<?= $this->flashMessenger()->render() ?>
+<?= $this->flashMessenger()->render(FlashMessenger::NAMESPACE_SUCCESS) ?>
```

Comment:
_Crap, why is it so complicated?_ This means that we need to extend the whole section to explain the usage with the `default` namespace and also the others.
I will create a suggestion for this.

Reply:
It is known because in the section of the manual, it is written that the method without an argument will pull all flash messages, but this does not work and is misleading.

The tutorial and many sections have such a problem, I have not come across such a problem in the symfony documentation, if the example is in the documentation, then it should work without assumptions

Reply:
@froschdesign If isn't complicated why laminas contributors add that doc section?

Reply:
Mistakes happen. If you find any further problems, we would be happy to hear from you.

Reply:
Will you merge my PR?
Is it my first PR to public repo)